numpy/add_newdocs.py

```diff
@@ -5421,40 +5421,19 @@ def luf(lamdaexpr, *args, **kwargs):
     """
     Functions that operate element by element on whole arrays.
 
-    To see the documentation for a specific ufunc, use np.info().  For
-    example, np.info(np.sin).  Because ufuncs are written in C
+    To see the documentation for a specific ufunc, use `np.info`.  For
```

Comment:
I think this needs to be double ticks to, or does it link correctly? (I ask since in astropy this would require `~numpy.info`; if the link does work, maybe pre-pend :func:)